hydrosdk/modelversion.py

```diff
@@ -335,86 +317,87 @@ class ModelStatus(Enum):
     Failed = "Failed"
 
 
-class Model(Metricable):
+class ModelVersion(Metricable):
     """
     Model (A model is a machine learning model or a processing function that consumes provided inputs
     and produces predictions or transformations
     https://hydrosphere.io/serving-docs/latest/overview/concepts.html#models)
+    ModelVersion represents one of the Model's versions.
     """
     BASE_URL = "/api/v2/model"
 
     @staticmethod
-    def find(cluster, name, version):
+    def find_by_name_id(cluster: Cluster, name: str, id_) -> 'ModelVersion':
```

Comment:
it's not id, but a numerical version of a model.

In order to avoid confusion I suggest renaming it to something like this (maybe?)
```suggestion
    def find(cluster: Cluster, name: str, version: int) -> 'ModelVersion':
```